components/safe_browsing/android/java/src/org/chromium/components/safe_browsing/BraveSafeBrowsingUtils.java

```diff
@@ -139,4 +140,35 @@ public static int safetyNetToSafeBrowsingJavaThreatType(int safetyNetThreatType)
                 return SafeBrowsingJavaThreatType.NO_THREAT;
         }
     }
+
+    // Priorities from most high to low
+    private static final int SAFETY_NET_THREAT_PRIORITIES[] = {
+        SafetyNetJavaThreatType.SOCIAL_ENGINEERING,
+        SafetyNetJavaThreatType.BILLING,
+        SafetyNetJavaThreatType.POTENTIALLY_HARMFUL_APPLICATION,
```

Comment:
I'd move POTENTIALLY_HARMFUL_APPLICATION to be second (i.e. between SOCIAL_ENGINEERING and BILLING).

Reply:
thanks, fixed at https://github.com/brave/brave-core/pull/26195/commits/cdb294ccfe8e78f31fe4404b4bd3264f885049a5